src/appkit.cs

```diff
@@ -25201,4 +25762,282 @@ interface NSSliderAccessoryBehavior : NSCoding, NSCopying
+		string FilterString { get; set; }
+	}
+	
+	[Mac (10,13)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface NSAccessibilityCustomRotorItemResult
+	{
+		[Export ("initWithTargetElement:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (NSAccessibilityElement targetElement);
+
+		[Export ("initWithItemLoadingToken:customLabel:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (INSSecureCoding itemLoadingToken, string customLabel);
+
+		[NullAllowed, Export ("targetElement", ArgumentSemantic.Weak)]
+		NSAccessibilityElement TargetElement { get; }
+
+		[NullAllowed, Export ("itemLoadingToken", ArgumentSemantic.Strong)]
+		INSSecureCoding ItemLoadingToken { get; }
+
+		[Export ("targetRange", ArgumentSemantic.Assign)]
+		NSRange TargetRange { get; set; }
+
+		[NullAllowed, Export ("customLabel")]
+		string CustomLabel { get; set; }
+	}
+
+	interface INSAccessibilityCustomRotorItemSearchDelegate {}
+
+	[Mac (10,13)]
+	[Protocol, Model]
+	[BaseType (typeof(NSObject))]
+	interface NSAccessibilityCustomRotorItemSearchDelegate
+	{
+		[Abstract]
+		[Export ("rotor:resultForSearchParameters:")]
+		[return: NullAllowed]
+		NSAccessibilityCustomRotorItemResult ResultForSearch (NSAccessibilityCustomRotor rotor, NSAccessibilityCustomRotorSearchParameters searchParameters);
+	}
+
+	interface INSAccessibilityElementLoading {}
+
+	[Mac (10,13)]
+	[Protocol, Model]
+	[BaseType (typeof(NSObject))]
+	interface NSAccessibilityElementLoading
+	{
+		[Abstract]
+		[Export ("accessibilityElementWithToken:")]
+		[return: NullAllowed]
+		NSAccessibilityElement GetAccessibilityElement (INSSecureCoding token);
+
+		[Export ("accessibilityRangeInTargetElementWithToken:")]
+		NSRange GetAccessibilityRangeInTargetElement (INSSecureCoding token);
+	}
+
+	interface INSCollectionViewPrefetching { }
+
+	[Mac (10,13)]
+	[Protocol, Model]
+	[BaseType (typeof(NSObject))]
+	interface NSCollectionViewPrefetching
+	{
+		[Abstract]
+		[Export ("collectionView:prefetchItemsAtIndexPaths:")]
+		void PrefetchItems (NSCollectionView collectionView, NSIndexPath[] indexPaths);
+
+		[Export ("collectionView:cancelPrefetchingForItemsAtIndexPaths:")]
+		void CancelPrefetchingForItems (NSCollectionView collectionView, NSIndexPath[] indexPaths);
+	}
+
+	delegate bool DownloadFontAssetsRequestCompletionHandler (NSError error);
+
+	[Mac (10,13)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface NSFontAssetRequest : INSProgressReporting
+	{
+		[Export ("initWithFontDescriptors:options:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (NSFontDescriptor[] fontDescriptors, NSFontAssetRequestOptions options);
+
+		[Export ("downloadedFontDescriptors", ArgumentSemantic.Copy)]
+		NSFontDescriptor[] DownloadedFontDescriptors { get; }
+
+		[Export ("progress", ArgumentSemantic.Strong)]
+		NSProgress Progress { get; }
+
+		[Export ("downloadFontAssetsWithCompletionHandler:")]
+		void DownloadFontAssets (DownloadFontAssetsRequestCompletionHandler completionHandler);
```

Comment:
So we can't [Async] delegates that return non-void, but the generator error was confusing:

https://bugzilla.xamarin.com/show_bug.cgi?id=57801